### ISSUE TITLE
A11y_mssql for VSCode_Messages Tab: When focus comes to link "Line1" jaws is reading column and row information twice and also not reading link (line1) name descriptively

### ISSUE THREAD

**Issue by InduPriya1805** [microsoft/vscode-mssql#1232]
GitHubTags: #A11y_mssqlforVSCode; #A11yMAS; #A11yTCS; #C+E-H; #DesktopApp; #Win10; #Jaws; #ScreenReader; #MAS1.3.1; #mssqlforVSCode

**Environment:**
Visual Studio Code – Insider (install from https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fcode.visualstudio.com%2Finsiders.&data=02%7C01%7C%7C00d9f7de52714b323f9908d607afe0c5%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636704849297354687&sdata=cmMqckjoNPGtCS3SZ00bCqCnFprKgiux%2BIjyvzgU0hE%3D&reserved=0)
Version: 1.34.0-insider (user Setup)
Mssql version: 1.6.0
OS Version: Microsoft Windows 10 Enterprise (10.0.17763 Build 17763)
Screen Reader: Jaws

**MAS ID Violated:** MAS 1.3.1

**Repro Steps:**
1. Open Visual Studio Code in your system
2. Navigate to "Extensions" and install "SQl Server (mssql)" extension
3. Press F1 and select MS SqL : Connect and connect to any sql server
4. "Write  code (SELECT *
FROM sys.objects;) in the new document and press F1 and select ""MS SQL : Execute Query"" "
5. Tab till "link1" present under messages tab and verify how jaws is reading the link "line1"

**Actual Result:**
When focus comes to link "Line1" jaws is reading column and row information twice and also not reading "line1" link name descriptively. Jaws is reading  as "Column 2 row1 Column 2 row1 line1 link"

**Expected Result:**
When focus comes to link "Line1" jaws should not read column information twice and also it should read the link name descriptively. When focus comes to "line1" link jaws should read as "Column 2 row1 started executed query at line1 link"

**User Impact:**
Screen reader users may get confused if it reads column and row information twice and also if jaws just reads as "line1 link" then screen reader users does not get any information such as what is the link about.
If it reads as "started executed query at line1 link" then it will be more descriptive.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

**Attachments:**
![Bug_1232_reading link information incorrectly](https://user-images.githubusercontent.com/42638450/56643096-8ab8b880-6696-11e9-850f-5df9a9cf11bf.png)

**Comment by anthonydresser** [comment]
What I am seeing looks correct.
![Untitled](https://user-images.githubusercontent.com/4324725/69758208-bb247180-1113-11ea-8308-2ad02dba0224.png)

If the desired effect is to read the entire text, we would need to make the whole text a link, but since the "line 1" is the linked part, that is what the screen reader is reading.

**Comment by BhanuPrakashGit** [comment]
GithubTag:#A11ySev2;

**Comment by Roopesh-T** [comment]
The announcement of the link Line1 is sufficient as per SMEs discussed, and the row and column info is being announced only once.
**Github Tag** : #Closed;

Version: 1.79.0-insider (user setup)

https://github.com/microsoft/vscode-mssql/assets/86601127/f09c2926-2176-493b-9d68-af7b4d442ded
